src/components/header/header.tsx

```diff
@@ -0,0 +1,24 @@
+import SearchForm from '../searchForm/searchForm';
+import GenreSelect from '../genreSelect/genreSelect';
+const Header = (props:GenreSelectProps) => {
+
+    const handleSearch = (e: React.FormEvent<HTMLFormElement>)=> {
+        e.preventDefault();
+          console.log(e);    
+        console.log("Search is Working !!");
+    
+      }
+      
+    return (
+        <>
+            <div className="header">
+                <SearchForm initialSearchQuery='What are you looking for ?' onSearch={handleSearch} />
```

Comment:
'What are you looking for ?'  Is an input placeholder, not the default param.
It can look as replaceable at first, but in reality - the value of the input will be the value you are searching for.
With this configuration, you will never receive the list of movies on the first load - there are no movies with ''What are you looking for ?' Title in the list.